Aliucord/src/main/java/com/aliucord/fragments/SelectDialog.kt

```diff
@@ -0,0 +1,84 @@
+package com.aliucord.fragments
+
+import android.view.View
+import android.view.ViewGroup
+import android.view.Gravity
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import androidx.annotation.NonNull
+import com.aliucord.Utils
+import com.aliucord.utils.DimenUtils
+import com.discord.app.AppDialog
+import com.lytefast.flexinput.R
+
+class SelectDialog() : AppDialog(Utils.getResId("widget_settings_language_select", "layout")) {
+    inner class Adapter(private val items: Array<String>) : RecyclerView.Adapter<Adapter.ViewHolder?>() {
+        @NonNull
+        override fun onCreateViewHolder(@NonNull parent: ViewGroup, viewType: Int): ViewHolder {
+            return ViewHolder(TextView(parent.getContext(), null, 0, R.i.UiKit_Settings_Item_Icon).apply {
+                layoutParams = ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
+            })
+        }
+
+        override fun onBindViewHolder(@NonNull holder: ViewHolder, position: Int) {
+            holder.bind(position)
+        }
+
+        override fun getItemCount() : Int {
+            return items.size
+        }
+
+        inner class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
```

Comment:
change type of this to TextView so you don't have to cast later

Reply:
i tried that before but it mysteriously didn't work

Reply:
sounds like you have skill issue lol because it does work